### ISSUE TITLE
Implement a function for file browsing, selecting, and simple processing

### ISSUE THREAD

**Issue by Nahrain1** [amosproj/amos2024ss07-updating-flash-boot-loader#5]
## User story
1. As a user
2. I want / need to select software files
3. So that I can process them with a C++ function to display their size and the first 16 bytes

## Acceptance criteria
* UI allows simple browsing through files
* Users can select a desired file from the available options.
* Selected files are passed to the C++ function 
* The file size and the 16 first bytes are displayed

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
